src/Faithlife.Math/Faithlife.Math.csproj

```diff
@@ -13,10 +13,6 @@
     <PackageProjectUrl>https://github.com/Faithlife/FaithlifeMath</PackageProjectUrl>
     <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/VersionHistory.md</PackageReleaseNotes>
     <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/RepoName/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/Faithlife/RepoName</PackageProjectUrl>
-    <PackageReleaseNotes>https://github.com/Faithlife/RepoName/blob/master/VersionHistory.md#released</PackageReleaseNotes>
```

Comment:
We need the fix in this line from https://github.com/Faithlife/CSharpTemplate/commit/5d141352f477c59bbc4a8eb5f34bd822d8d3dd6d

It has to be merged with the right repo name on line 14.